test-utils/runtime-tester/src/fuzzing.rs

```diff
@@ -344,14 +403,22 @@ pub enum Function {
 
 impl Scope {
     fn from_seeds(seeds: &[String]) -> Self {
-        let accounts = seeds.iter().map(|id| Account::from_id(id.clone())).collect();
+        let accounts: Vec<Account> = seeds.iter().map(|id| Account::from_id(id.clone())).collect();
+        let account_id_to_idx = {
+            let mut result = HashMap::new();
+            for i in 0..accounts.len() {
+                result.insert(accounts[i].id.clone(), i);
+            }
+            result
+        };
```

Comment:
Seems this can be shorteneed with the help of accounts.iter().enumerate().map(...).collect()